demisto_sdk/commands/download/downloader.py

```diff
@@ -412,8 +410,8 @@ def fetch_custom_content(self) -> bool:
                     path.write_text(string_to_write)
 
                 except Exception as e:
-                    print(f"encountered exception {type(e)}: {e}")
-                    print("trying to write with encoding=utf8")
+                    logger.info(f"encountered exception {type(e)}: {e}")
+                    logger.info("trying to write with encoding=utf8")
```

Comment:
```suggestion
                    logger.exception(f"encountered exception, trying to write with encoding=utf8")
```

I wrote this code, it's okay